src/main/kotlin/org/dropProject/controllers/ApplicationContextListener.kt

```diff
@@ -155,6 +169,7 @@ class ApplicationContextListener(val assignmentRepository: AssignmentRepository,
 
             if (connected) {
 
+                /*
```

Comment:
Why is this commented?

Reply:
As agreed, I will uncomment this code and do a new PR